projects/lc-manual.diff

```diff
@@ -527,475 +527,7 @@ index 5d47217..5a80aff 100644
-                 </core:content>
-@@ -5156,7 +5166,7 @@ return objid
-         </target>
-         <complexParameter name="script">
-             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
--def objid = 'LandCoverUnit_' + id&#13;
-+def objid = 'LandCoverUnit_' + gml_id&#13;
- &#13;
- withTransformationContext {&#13;
-   def c = _.context.collector(it)&#13;
-@@ -5169,7 +5179,7 @@ return objid
-         <parameter value="false" name="variablesAsInstances"/>
-         <documentation>Da alle FeatureTypes, die für das Mapping verwendet werden, von AA_Objekt erben, wird das Mapping der id auf dieser Ebene durchgeführt. IDs der erzeugten LandCoverUnits werden mit Hilfe des GroovyScripts gesammelt um sie in einer niedriger priorisierten Transformation dem zugehörigen LandCoverDataset zuzuweisen</documentation>
-         <annotation type="message">
--            <core:message xmlns:core="http://www.esdi-humboldt.eu/hale/core" category="migration">
-+            <core:message xmlns:core="http://www.esdi-humboldt.eu/hale/core" category="migration" dismissed="true">
-                 <core:content>
-                     <core:text xml:space="preserve">The cell's source entities have been replaced, the new entities may have a different structure and/or names and may require updating the Groovy script accordingly</core:text>
-                 </core:content>
-@@ -5193,7 +5203,7 @@ return objid
-         </target>
-         <complexParameter name="script">
-             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
--def objid = 'LandCoverUnit_' + id&#13;
-+def objid = 'LandCoverUnit_' + gml_id&#13;
- &#13;
- withTransformationContext {&#13;
-   def c = _.context.collector(it)&#13;
-@@ -5206,7 +5216,7 @@ return objid
-         <parameter value="false" name="variablesAsInstances"/>
-         <documentation>Da alle FeatureTypes, die für das Mapping verwendet werden, von AA_Objekt erben, wird das Mapping der id auf dieser Ebene durchgeführt. IDs der erzeugten LandCoverUnits werden mit Hilfe des GroovyScripts gesammelt um sie in einer niedriger priorisierten Transformation dem zugehörigen LandCoverDataset zuzuweisen</documentation>
-         <annotation type="message">
--            <core:message xmlns:core="http://www.esdi-humboldt.eu/hale/core" category="migration">
-+            <core:message xmlns:core="http://www.esdi-humboldt.eu/hale/core" category="migration" dismissed="true">
-                 <core:content>
-                     <core:text xml:space="preserve">The cell's source entities have been replaced, the new entities may have a different structure and/or names and may require updating the Groovy script accordingly</core:text>
-                 </core:content>
-@@ -7459,6 +7469,78 @@ return objid
```

Comment:
Maybe I am confused again with the diff of diffs but if I get it right, the lines 530 - 998 of the diff file are removed. Are these diffs really not needed any longer?

Reply:
I checked the corresponding alignment file in the adv-inspire-alignments folder and couldn't find occurrences of
```
def objid = 'LandCoverUnit_' + id&#13;
```
in there. So I assume that those have been removed from there and therefore the manual changes relating to them are also no longer needed.